test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -200,7 +201,7 @@ export class DisplayTestApp {
     socketUrl.protocol = "ws";
     socketUrl.pathname = [...socketUrl.pathname.split("/"), "ipc"].filter((v) => v).join("/");
 
-    const opts = {
+    const opts: any = {
```

Comment:
If you added `: any` because compiler complained, there's probably a real error. What was the error?

Reply:
Thanks for pointing that out. The problem was that the new line on 235 where I was adding the electron auth client as `opts.iModelApp.authorizationClient` was throwing an error since authorizationClient wasn't a defined property on the object. So I just now changed it to where the opts object is typed as `ElectronAppOpts | LocalHostIpcAppOpts` so that it knows that is a real property. The side effects of this are now all the properties are viewed as possibly undefined, so I had to add a conditional check for the electron startup and browser startup sections when referencing properties since non-null assertions are not advised. If you have a better solution let me know.

Reply:
What you have now is fine. Alternative would have been to initialize authorizationClient before opts and do `opts = { iModelApp: { authorizationClient } }`.